projects/reservoir/fv3/era_process.py

```diff
@@ -52,16 +54,22 @@ def add_arguments():
         "--out_path",
         help="path to save regridded data, should include whether train/val/test",
     )
-    parser.add_argument(
-        "--train_val_test", help="whether you are creating train, val or test data"
-    )
     parser.add_argument(
         "--start_date", help="start date as a string in the format of YYYY-MM-DD"
     )
     parser.add_argument(
         "--end_date", help="end date as a string in the format of YYYY-MM-DD"
     )
     parser.add_argument("--variables", nargs="+", help="list of variables to process")
+    parser.add_argument(
+        "--fill_value_sst", default=291, type=float, help="value to fill sst nans"
+    )
+    parser.add_argument(
```

Comment:
Can be removed since now the reservoir training can save a mask array to deal with unwanted inputs. However since that would require additional changes to this PR, just leave this for now and add a TODO comment above it mentioning that it can be removed in a subsequent PR.